### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by meowgorithm**
Minor documentation updates in anticipation of the next release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -300,37 +299,40 @@ someStyle.MaxWidth(5).MaxHeight(5).Render("yadda yadda")
...
+in a server-client situation.
 
 ```go
-renderer := lipgloss.NewRenderer(
-    lipgloss.WithColorProfile(termenv.ANSI256),
-    lipgloss.WithDarkBackground(true),
-)
+func myLittleHandler(sess ssh.Session, pty ssh.Pty) {
+    // Create a renderer for the client.
+    renderer := lipgloss.NewRenderer(lipgloss.WithOutput(pty))
```
Comment (aymanbagabas) [inline README.md:324]:
```suggestion
    renderer := lipgloss.NewRenderer(sess)
```

File: README.md [hunk 2]
```diff
@@ -300,37 +299,40 @@ someStyle.MaxWidth(5).MaxHeight(5).Render("yadda yadda")
...
+particularly important when you want to render to different outputs and
+correctly detect the color profile and dark background status for each, such as
+in a server-client situation.
 
 ```go
-renderer := lipgloss.NewRenderer(
-    lipgloss.WithColorProfile(termenv.ANSI256),
-    lipgloss.WithDarkBackground(true),
-)
+func myLittleHandler(sess ssh.Session, pty ssh.Pty) {
```
Comment (aymanbagabas) [inline README.md:322]:
```suggestion
func myLittleHandler(sess ssh.Session) {
```

File: examples/ssh/main.go [hunk 3]
```diff
@@ -14,6 +23,41 @@ import (
...
+	green         lipgloss.Style
+	yellow        lipgloss.Style
+	blue          lipgloss.Style
+	magenta       lipgloss.Style
+	cyan          lipgloss.Style
+	gray          lipgloss.Style
+}
+
+// Create new styles against a given renderer.
+func makeStyles(r *lipgloss.Renderer) styles {
```
Comment (aymanbagabas) [inline examples/ssh/main.go:43]: `r` not used

**Review by aymanbagabas — COMMENTED** [review]
Some nits and bits.
Require https://github.com/charmbracelet/lipgloss/pull/174

File: examples/ssh/main.go (hunk 3 above)
Comment (meowgorithm) [reply examples/ssh/main.go:43]: It's being used in the return to create styles no?

Comment (aymanbagabas) [reply examples/ssh/main.go:43]: oh sorry missed that 😄

**Review by aymanbagabas — APPROVED** [review]

**Pull request merged**